integration/package-operator/objecttemplate_test.go

```diff
@@ -446,7 +446,8 @@ data:
 				upatedCM := obj.(*corev1.ConfigMap)
 				return upatedCM.Data["test"] == secretValue, nil
 			},
-			wait.WithTimeout(60*time.Second),
+			// For optional source the default retryAfter duration in the controller is 60s.
+			wait.WithTimeout(100*time.Second),
```

Comment:
We could reduce the retryAfter duration to something like 1 or 2 seconds to shave off almost a full minute of the int tests - that'd be  a nice bonus that we get from having that duration configurable.

WDYT?

Reply:
👍